src/styles/style-helpers.js

```diff
@@ -64,6 +64,24 @@ export const svgMega = ({ theme }) => size(theme.iconSizes.mega);
  * Utilities
  */
 
+export const centerAlign = direction => {
+  if (direction === 'vertical') {
+    return `
+      top: 50%;
+      transform: translateY(-50%);
+    `;
+  }
+  if (direction === 'horizontal') {
+    return `
+      left: 50%;
+      transform: translateX(-50%);
+    `;
+  }
+  // eslint-disable-next-line no-console
+  console.warn(`You passed an invalid direction: ${direction}`);
```

Comment:
Do you want to keep it like this? I think this should be limited to develop mode.

Reply:
Develop mode where? In circuit-ui? Or when used in a project like the dashboard? 

In other words, should this be stripped when circuit-ui is compiled for npm?

Reply:
Will rewrite so the console statement isn't necessary at all.